src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookInstance.ts

```diff
@@ -1163,7 +1157,7 @@ export class PositronNotebookInstance extends Disposable implements IPositronNot
 	pasteCellsAbove(): void {
 		const selection = this.selectionStateMachine.getSelectedCells();
 		if (selection.length > 0) {
-			const firstSelectedIndex = this.cells.get().indexOf(selection[0]);
+			const firstSelectedIndex = selection[0].index;
```

Comment:
Should we add a helper for this logic in the selectionStateMachine or similar? I feel we do `selection[0]` a lot.